### PR TITLE
feat: add color presets for users to choose from

### PR DESCRIPTION
**Pull request by simschla**
I've always struggled in choosing good color combinations for my project label. So I went ahead and created a preset palette for users to choose from.

Light colors:
SCR-20230719-iuxs

Dark colors:
SCR-20230719-ivbs

Let me know what you think!

### CODE REVIEW

**Comment by drinchev** [comment]
Hi, it looks awesome. 

I will test the commit early next week and hopefully we can merge. 

Thanks a lot!

File: src/main/java/com/drinchev/projectlabel/resources/ui/ColorPaletteElement.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            g2d.drawArc(x, y, diameter, diameter, 0, 360);
+        }
+        g2d.dispose();
+    }
+
+    public static void main(String[] args) {
+        JFrame frame = new JFrame();
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.setLayout(new BorderLayout());
+        ColorPaletteElement paletteElement = new ColorPaletteElement(new ColorPair("#6369D1", "#F5F5F5", "Test"));
```
Comment (drinchev) [inline src/main/java/com/drinchev/projectlabel/resources/ui/ColorPaletteElement.java]: Sorry, I'm trying to get the logic behind this main method. But what does it really do? I see that the color pair is assigned in the constructor method, so I'm really not sure why we need `main` here.

**Comment by drinchev** [comment]
Thanks a lot. Looks and feel great.

Can you also update `gradle.properties` version to `1.3.0` and the `CHANGELOG.md`. 

This way I will be able to release it without making another commit :P

File: src/main/java/com/drinchev/projectlabel/resources/ui/ColorPaletteElement.java (hunk 1 above)
Comment (simschla) [reply src/main/java/com/drinchev/projectlabel/resources/ui/ColorPaletteElement.java]: I apologize for the confusion. That's debug code I've used while designing the UI elements. I'll remove that!

**Comment by simschla** [comment]
I'll gladly update that 😀
I'm a bit swamped this week, so will probably clean up and finalize the PR next week if that is ok with you!

**Comment by simschla** [comment]
Thank you for your patience. I've finished cleaning up: Removed the unnecessary main method, bumped the version in gradle.properties and added an entry in the changelog.

**Review by drinchev — APPROVED** [review]

**Comment by drinchev** [comment]
Hi,

The plugin is pending approval by JetBrains 🥳

**Pull request merged**